ResourceStatusSystem/Command/FreeDiskSpaceCommand.py

```diff
@@ -0,0 +1,115 @@
+''' FreeDiskSpaceCommand
+
+  The Command gets the free space that is left in a DIRAC Storage Element
+
+'''
+
+from datetime                                                   import datetime
+from DIRAC                                                      import S_OK, S_ERROR, gConfig, gLogger
+from DIRAC.ResourceStatusSystem.Command.Command                 import Command
+from DIRAC.Core.DISET.RPCClient                                 import RPCClient
+from DIRAC.ResourceStatusSystem.Utilities                       import CSHelpers
+
+__RCSID__ = '$Id:  $'
+
+
+class FreeDiskSpaceCommand( Command ):
+  '''
+  Uses diskSpace method to get the free space
+  '''
+
+  def __init__( self, args = None ):
+
+    super( FreeDiskSpaceCommand, self ).__init__( args )
+
+    self.rpc = None
+    self.rsClient = None
+
+  def getUrl(self, SE, protocol = None ):
+    """
+    Gets the url of a storage element from the CS.
+    If protocol is set, then it is going to fetch the
+    url only if it uses the given protocol.
+
+    :param SE: String
+    :param protocol: String
+    :return: String
+    """
+    attributes = [ "Protocol", "Host", "Port", "Path"]
+
+    result = ""
+    for attribute in attributes:
+      res = gConfig.getValue( "/Resources/StorageElements/%s/AccessProtocol.1/%s" % ( SE, attribute ) )
+
+      if protocol:
+
+        # Not case-sensitive
+        protocol = protocol.lower()
+
+        if attribute is "Protocol" and res != protocol:
+          result = None
+          break
+
+      if attribute is "Protocol":
+        result += res + "://"
+      elif attribute is "Port":
+        result += ":" + res
+      else:
+        result += res
+
+    return result
+
+  def _prepareCommand( self ):
+    '''
+      FreeDiskSpaceCommand requires one argument:
+      - element : Resource
+
+    '''
+
+    if 'element' not in self.args:
+      return S_ERROR( '"element" not found in self.args' )
+    element = self.args[ 'element' ]
```

Comment:
`element` is in RSS usally either "Site" or "Resource". I think what you are looking for here is `name` (the name of the resource)